src/pip/_internal/cli/cmdoptions.py

```diff
@@ -494,9 +494,11 @@ def only_binary():
     '--platform',
     dest='platform',
     metavar='platform',
+    action='append',
     default=None,
-    help=("Only use wheels compatible with <platform>. "
-          "Defaults to the platform of the running system."),
+    help=("Only use wheels compatible with <platform>. Defaults to the "
+          "platform of the running system. Use multiple options to specify "
```

Comment:
How about *Use this option multiple times …*?

Reply:
👍Done (for both `--abi` and `--platform`).